plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/BurpReader.java

```diff
@@ -131,45 +131,45 @@ static int cweLookup(String id) {
             case "2098944":
                 return CweNumber.CSRF;
             case "3146240":
-                return 918; // External service interaction (DNS)
+                return CweNumber.SERVER_SIDE_REQUEST_FORGERY; // External service interaction (DNS)
             case "4194560":
                 return CweNumber.DONTCARE; // Referer Dependent Response
             case "4194576":
                 return CweNumber.DONTCARE; // X-Forwarded-For header dependency
-            case "4197376":
-                return 20; // Input returned in response (reflected)
-            case "4197632":
-                return 20; // Suspicious input transformation (reflected)
+            case "4197376": // Input returned in response (reflected)
+            case "4197632": // Suspicious input transformation (reflected)
+                return CweNumber.IMPROPER_INPUT_VALIDAITON;
             case "5243392":
                 return CweNumber.INSECURE_COOKIE;
             case "5244416":
-                return 9998; // Cookie without HttpOnly flag set - There is no CWE defined for this
-                // weakness
-            case "5245344":
-                return 1021; // Clickjacking
-            case "5245360":
-                return 16; // Browser cross-site scripting filter disabled
+                return CweNumber.COOKIE_WITHOUT_HTTPONLY; // Cookie without HttpOnly flag set
+            case "5245344": // Clickjacking
+                return CweNumber.IMPROPER_RESTRICTION_OF_UI_LAYERS;
+            case "5245360": // Browser cross-site scripting filter disabled
+                return CweNumber.CATEGORY_CONFIGURATION;
             case "5245952":
                 return CweNumber
                         .DONTCARE; // Ajax request header manipulation (DOM-based) - Map to nothing
                 // right
             case "5247488":
                 return CweNumber
                         .DONTCARE; // DOM Trust Boundary Violation - Map to nothing right now.
-            case "6291968":
-                return 200; // Information Disclosure - Email Address Disclosed
-            case "6292736":
-                return 200; // Information Disclosure - Credit Card # Disclosed
+            case "6291968": // Information Disclosure - Email Address Disclosed
+            case "6292736": // Information Disclosure - Credit Card # Disclosed
+                return CweNumber.EXPOSURE_SENSITIVE_TO_UNAUTHORIZED_USER;
             case "7340288":
-                return 525; // Information Exposure Through Browser Caching-Cacheable HTTPS Response
+                return CweNumber
+                        .SENSITIVE_INFORMATION_IN_BROWSER_CACHE; // Information Exposure Through
+                // Browser Caching-Cacheable HTTPS
+                // Response
```

Comment:
Can you merge the above 2 lines into a single line comment?

Reply:
Already fixed